IPython/extensions/cythonmagic.py

```diff
@@ -8,36 +10,17 @@
 """
 #-----------------------------------------------------------------------------
 # Copyright (C) 2010-2011, IPython Development Team.
-#
-# Distributed under the terms of the Modified BSD License.
-#
-# The full license is in the file COPYING.txt, distributed with this software.
 #-----------------------------------------------------------------------------
 
-from __future__ import print_function
-
-import IPython.utils.version as version
-
-try:
-    import Cython
-except:
-    Cython = None
-
-try:
-    from Cython.Build.IpythonMagic import CythonMagics
-except :
-    pass
-
+import warnings
 
 ## still load the magic in IPython 3.x, remove completely in future versions.
 def load_ipython_extension(ip):
     """Load the extension in IPython."""
     
-    print("""The Cython magic has been moved to the Cython package, hence """)
-    print("""`%load_ext cythonmagic` is deprecated; please use `%load_ext Cython` instead.""")
+    warnings.warn("""
+The Cython magic has been moved to the Cython package, hence %load_ext cythonmagic` is deprecated; 
+Please use `%load_ext Cython` instead.""")
```

Comment:
It's not deprecated if it doesn't work anymore. I think this is fine if you just remove the "hence..." clause.